crates/nu-cli/src/commands/histogram.rs

```diff
@@ -63,7 +63,7 @@ impl WholeStreamCommand for Histogram {
             },
             Example {
                 description: "Get a histogram for a list of numbers",
-                example: "echo [1 2 3 1 1 1 2 2 1 1] | histogram",
+                example: "echo [1 2 3 1 1 1 2 2 1 1] | wrap values | histogram values",
                 result: None,
```

Comment:
Since you mentioned the `result` for testing in the ticket: I ended up adding a different example including a row result: https://github.com/nushell/nushell/pull/2256/files#diff-4877b0697e6002708a4ac3ed35d8f792R52 That could work here, too.

Reply:
This is a good suggestion, however, no matter what I try, it always fails:

```rust
// Setup for the test of example "Get a histogram for a list of numbers"
let mut row_1 = IndexMap::new();

row_1.insert("values".to_string(), Value::from("1"));
row_1.insert("count".to_string(), UntaggedValue::int(6).into());
row_1.insert("frequency".to_string(), Value::from("****************************************************************************************************"));

let mut row_2 = IndexMap::new();

row_2.insert("values".to_string(), Value::from("2"));
row_2.insert("count".to_string(), UntaggedValue::int(3).into());
row_2.insert(
    "frequency".to_string(),
    Value::from("**************************************************"),
);

let mut row_3 = IndexMap::new();

row_3.insert("values".to_string(), Value::from("3"));
row_3.insert("count".to_string(), UntaggedValue::int(1).into());
row_3.insert("frequency".to_string(), Value::from("****************"));

let result = Some(vec![
    UntaggedValue::row(row_1).into(),
    UntaggedValue::row(row_2).into(),
    UntaggedValue::row(row_3).into(),
]);
```

```shell
---- commands::histogram::tests::examples_work_as_expected stdout ----
thread 'commands::histogram::tests::examples_work_as_expected' panicked at 'failed to run example: ShellError { error: UntaggedRuntimeError { reason: "Could not load command: run_external" }, cause: None }', crates/nu-cli/src/examples.rs:29:26
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
I'm okay with skipping adding an example result for now while we work on a better way of adding results. That version is pretty difficult to read, so it's hard to confirm if you got the right answer.

Reply:
Maybe I should get @thegedge to weigh in

Reply:
Right.  I spend quite a long time trying to figure out what was wrong, over and over, and never figured it out.

Reply:
Yeah, I added the `Option` type on example results exactly because it's hard to construct these. I'm a-o-k switching to `None`.